### PR TITLE
Macaria - Merchant Show Page

### PR DESCRIPTION
**Pull request by mdove92**
Created temporary Merchant Show page. Has overlapping issues with application.html.erb

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -42,7 +42,8 @@
         </div>
       </nav>
     </header>
-
+<section>
```
Comment (tofuandeve) [inline app/views/layouts/application.html.erb:45]: If we decided to fix the layout later then we can just remove this section tag and put a task Trello board backlog. Or if we want to keep this section tag then we should change the indentation of this part (line 45 to 47). I'm okay with both though.

**Review by tofuandeve — APPROVED** [review]
Great in general. I left one minor comment but I'm okay with any decision you make on that part!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -28,7 +28,7 @@
           <ul class="navbar-nav navbar-right">
             <% if session[:merchant_id]%>
               <li class="nav-item app-header__nav_item">
-                <%= link_to "Logged in as #{Merchant.find_by(id:session[:merchant_id]).username}", root_path, class: "nav-link" %>
+                <%= link_to "Logged in as #{Merchant.find_by(id:session[:merchant_id]).username}", merchant_path(session[:merchant_id]), class: "nav-link" %>
```
Comment (emaust) [inline app/views/layouts/application.html.erb:31]: Love this decision. A+! -Raccoon

File: app/views/merchants/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+</h2>
+<p>Joined site<%= "@merchant.created_at" %></p>
+
+<section class="merchant-products__container">
+  <h3 class="merchant-products__header">Products</h3>
+  
+<table class="table"> 
+<th> Product Name</th>
+<th> Current Inventory</th>
+<th> Amount Sold</th>
```
Comment (emaust) [inline app/views/merchants/show.html.erb:13]: Amount sold isn't a parameter, but is a good idea! Trello it?

File: app/views/merchants/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+<th> Category</th>
+<th> Price</th>
+<th> Date Listed</th>
```
Comment (emaust) [inline app/views/merchants/show.html.erb:16]: Does this automatically get tracked? It isn't in the seed file but idk if that means anything

File: app/views/merchants/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+
+
+
+
+<tr>
+<td><%= "Mayonnaise" %></td>
```
Comment (emaust) [inline app/views/merchants/show.html.erb:23]: DELICIOUS -Raccoon

**Review by emaust — APPROVED** [review]
This is great!

**Review by michaela260 — APPROVED** [review]
Yay! Looks good to me :)

**Pull request merged**